backend/apps/accounts/views.py

```
@@ -1098,3 +1101,39 @@ def get(self, request):
+    def post(self, request):
+        return Response({"status": "avatar uploaded stub"})
+
+
+class ShopStreakFreezeView(APIView):
+    permission_classes = [IsAuthenticated]
+    def post(self, request):
+        return Response({"status": "streak freeze purchased stub"})
+
+
+class PasswordResetValidateTokenView(APIView):
+    permission_classes = [permissions.AllowAny]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)